src/sentry/integrations/msteams/integration.py

```diff
@@ -77,18 +79,17 @@ def get_pipeline_views(self):
     def build_integration(self, state):
         data = state[self.key]
         team_id = data["team_id"]
+        team_name = data["team_name"]
         service_url = data["service_url"]
 
+        # TODO: add try/except for request errors
         token_data = get_token_data()
-        access_token = token_data["access_token"]
 
-        client = MsTeamsPreInstallClient(access_token, service_url)
-        team = client.get_team_info(team_id)
```

Comment:
Realized we actually get the team name in the initial webhook so no need to make this extra call once we add it to state